cvat/apps/dataset_manager/project.py

```diff
@@ -139,7 +138,7 @@ def export(self, dst_file: str, exporter: Callable, host: str='', **options):
         )
 
         temp_dir_base = self.db_project.get_tmp_dirname()
-        os.makedirs(temp_dir_base, exist_ok=True)
```

Comment:
Why these `makedirs` were removed? They have to create parent dirs. `TemporaryDirectory` fails, if these dirs do not exist.

Reply:
`makedirs` calls were moved into `model.get_tmp_dirname` method